persistence/sql/sql_process_store.go

```diff
@@ -50,7 +50,7 @@ func (p sqlProcessStoreImpl) Close() error {
 	return p.session.Close()
 }
 
-func (p sqlProcessStoreImpl) CleanUpTasksForTest(ctx context.Context, shardId int32) error {
+func (p sqlProcessStoreImpl) CleanUpImmediateTasksForTest(ctx context.Context, shardId int32) error {
```

Comment:
Looks like a typo?

Reply:
Oh yeah I thought this method only cleans up the immediate_task table. After checking I found this method cleans up both task tables. Renamed it back to `CleanUpTasksForTest `.